babel.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "corejs": 3,
+      "useBuiltIns": "entry",
```

Comment:
If you take a look at the [docs](https://babeljs.io/docs/en/babel-preset-env#usebuiltins), you will notice that the following lines need to be available in the top-level application entrypoint for `entry` mode:

```typescript
import "core-js/stable"
import "regenerator-runtime/runtime"
```

Reply:
Adding these in next commit, thanks for the catch!